config/secrets.json.example

```diff
@@ -3,19 +3,13 @@
     "consumer_key": "Enter Consumer Key",
     "consumer_secret": "Enter Consumer Secret",
     "access_token": "Enter Access Token",
-    "access_token_secret": "Enter Access Token Secret",
-    "configured": false,
-    "on": false
+    "access_token_secret": "Enter Access Token Secret"
```

Comment:
The settings can still be configured in the secrets.json. And we still use that file as storage for the settings, even if modified online